backend/routes/user.js

```diff
@@ -0,0 +1,124 @@
+const express = require('express');
+const router = express.Router();
+const gravatar = require('gravatar');
+const bcrypt = require('bcryptjs');
+const jwt = require('jsonwebtoken');
+const passport = require('passport');
+const validateRegisterInput = require('../validation/register');
+const validateLoginInput = require('../validation/login');
+
+const User = require('../models/User');
+
+router.post('/register', function(req, res) {
```

Comment:
too long callback function

For example, create some controls folder, create register controller and move there callback ib separate function.
+ split function. For example,
- call to backend (if email exist)
- register bussines logic.

Reply:
done